test/fixtures/content-explorer/root-folder.json

```diff
@@ -0,0 +1,682 @@
+{
```

Comment:
Any reason why we have to mock the network? The explorer should be able to return all of this info with our public token right?

Reply:
The default page doesn't have the upload and share buttons, so mocking the network was the only way I could get them to appear.  Can I get the upload and share buttons to appear without mocking?

Reply:
You should be able to intercept the server response and decorate it with the specific mocks you need rather than mocking the entire response.

Reply:
Take a look at the [Response Functions section](https://docs.cypress.io/api/commands/route.html#With-Stubbing) of the Cypress docs on `cy.route`.